### DISCUSSION TITLE
How to set different RSA encryption key for different folder ?

### DISCUSSION THREAD

**Discussion by SimonYang-CS** [juicedata/juicefs#2066] [Q&A] [answered]
Hi team,

I read [encrypt document ](https://github.com/juicedata/juicefs/blob/main/docs/en/security/encrypt.md), seems juicefs  only support one RSA Key pair.

Assume we have three tenants and mount on three folders:

```text
/mount/tenant-001/
                 data-folder-for-tenant-1-x...
/mount/tenant-002/
                 data-folder-for-tenant-2-y...
/mount/tenant-003/
                 data-folder-for-tenant-3-z...
```

How to set different  RSA encryption key for different  folder using Juicefs ?

**Comment by davies** [comment, marked as answer]
We can't right now. Maybe you need to create multiple volumes for different tenants, or use EcryptFS on top of JuiceFS.

**Reply by SimonYang-CS** [reply]
@davies , thanks for suggestions.
